package.json

```diff
@@ -13,7 +13,8 @@
     "lint": "tslint -p ./tsconfig.json",
     "lint:fix": "tslint -p ./tsconfig.json --fix",
     "serve": "ws --spa index.html --rewrite '/api/* -> http://localhost:3000/$1' -d ./dist",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npm run test:unit",
+    "test:unit": "ts-node tests/unit/index.ts",
```

Comment:
May need to add the package `ignore-styles` for testing our actual components with ts-node. Then add `-r ignore-styles` here.

Reply:
### PR Review Checklist

* [x] **Have you tested the changes?**
* [x] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [x] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [x] **Are the added/edited files named appropriately and located correctly in the project directory structure?**